.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: go
+go:
+  - 1.5
+before_install:
+  - sudo apt-get update
+  - sudo apt-get install -qq libgl1-mesa-dev xorg-dev
+install:
+  - go get golang.org/x/tools/cmd/vet
+script:
+  - go get -t -v ./v3.1/...
+  - diff -u <(echo -n) <(gofmt -d -s .)
```

Comment:
One more thing, the `diff <(foo) <(bar)` thing is called [process substitution](https://en.wikipedia.org/wiki/Process_substitution). Bash supports it.